appveyor.yml

```diff
@@ -20,28 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      VC_VERSION: 9
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      VC_VERSION: 9
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      VC_VERSION: 10
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      VC_VERSION: 10
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      VC_VERSION: 14
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      VC_VERSION: 14
-      CONDA_PY: 35
```

Comment:
Have marked the Windows packages as broken. Will add a new PR to fix this momentarily.

<img width="1554" alt="screen shot 2016-09-15 at 10 49 30" src="https://cloud.githubusercontent.com/assets/3019665/18554401/335b11b6-7b32-11e6-8245-bef6bf8006cf.png">